index.html

```diff
@@ -51,7 +51,7 @@
   </main>
   <footer class="page-footer footer-light transparent">
     <div class="row">
-      <div class="col m6 offset-m3 center"> <img class="footer-logo" src="https://api.coala.io/en/latest/_static/images/coala_logo.svg"> <span class="footer-title">{{ site.organization }}</span> </div>
+      <div class="col m6 offset-m3 center"> <img class="footer-logo" src="https://api.coala.io/en/latest/_static/images/coala_logo.svg" alt="coala"> <span class="footer-title">{{ site.organization }}</span> </div>
```

Comment:
This change should be in the other commit, right? https://github.com/coala/projects/pull/333/commits/c579dc90bdcc25ed68ca30b5e36452725e65ce46